src/app/reducers/firebaseUserReducer.js

```diff
@@ -0,0 +1,24 @@
+import * as ActionTypes from '../actions/types';
+
+export default function (state = null, action) {
+  switch (action.type) {
```

Comment:
Yes! This was some boilerplate I copied, I'll squash it down, but I think to be safe I'll still specify that it's one of those action types and return state for default